project.clj

```diff
@@ -104,7 +105,8 @@
                                      "test/resources/hooks/hook-example-0.1.0-SNAPSHOT.jar"]}
              :prepush {:plugins [[yogsototh/lein-kibit "0.1.6-SNAPSHOT"]
                                  [lein-bikeshed "0.3.0"]]}}
-  :plugins [[lein-shell "0.5.0"]]
+  :plugins [[lein-shell "0.5.0"]
+            [perforate "0.3.4"]]
```

Comment:
For now, I only created one benchmark test, but I imagine that yes, we certainly want to focus only on certain benchmarks. So I'll add this.